app/forms/teacher_interface/new_session_form.rb

```diff
@@ -0,0 +1,13 @@
+class TeacherInterface::NewSessionForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attribute :email, :string
+  attribute :create_or_sign_in, :string
+
+  validates :email, presence: true, valid_for_notify: true
```

Comment:
I had some further thoughts about this validation, and I think we could do more here.

```rb
validates :create_of_sign_in, presence: true
```

This checks that the user at least checks one of the radio buttons, so we can show an error if they press submit without having checked one of the radio buttons.

```rb
validates :email, presence: true, valid_for_notify: true, if: -> { !create? }
```

The `if: -> { !create? }` block means that this validation will only run, if `create_of_sign_in == "sign_in"` (i.e. that the user has said they want to sign in).

Once this validation is in place, we could then call `.valid?` in the controller before calling `.create?` and `redirect_to : eligibility_interface_countries`.